designcompose/src/main/java/com/android/designcompose/Utils.kt

```diff
@@ -1344,22 +1345,24 @@ internal fun Background.asBrush(
         is Background.Image -> {
             val backgroundImage = this
             val imageTransform = backgroundImage.transform.asSkiaMatrix()
-            backgroundImage.res_name.orElse(null)?.let {
-                val resId =
-                    appContext.resources.getIdentifier(it, "drawable", appContext.packageName)
-                if (resId != Resources.ID_NULL) {
-                    return Pair(
-                        RelativeImageFill(
-                            image = BitmapFactory.decodeResource(appContext.resources, resId),
-                            imageDensity = density,
-                            displayDensity = density,
-                            imageTransform = imageTransform,
-                            scaleMode = backgroundImage.scale_mode
-                        ),
-                        backgroundImage.opacity
-                    )
-                } else {
-                    Log.w(TAG, "No drawable resource $it found")
+            if (useLocalDrawableRes != false) {
```

Comment:
Do you know if getting the `useLocalDrawableRes` value from `DebugNodeManager` directly here would work? I'm just trying to avoid adding more parameters to our functions that already have a lot of parameters. I think doing it here wouldn't trigger a recomposition, but maybe that's ok (and good?). This isn't a big deal since this is just a debugging feature, but we do seem to keep adding parameters to functions.

Reply:
Squoosh works. It might because when we toggle the design switcher toggle, it draws the entire tree.
But the non-squoosh does not work upon toggling the checkbox, but it will render correctly the next time it draws.
Would you like to do this way? I can revert the parameter changes if so.

Reply:
Interestingly, I did a test and found that for text localization, the opposite is true: squoosh will not update immediately, but non-squosh does.

To fix this, I think you can use `MutableState<Boolean>` instead of `LiveData`. Then the declaration would be:
```
    private val useLocalStringRes: MutableState<Boolean> = mutableStateOf(true)
```
and the getter would return a `State<Boolean>`.

Reply:
I'm changing to use the State<Boolean>. For the squoosh text not updating upon toggling the checkbox in design switcher, it might be related to the text cache squoosh is using. Let me check...

Reply:
Done.